spec/ttml3.xml

```diff
@@ -1410,6 +1427,24 @@ consist of one or more <loc href="#terms-inline-area">inline areas</loc>.</p>
 <p>Any of the element types defined by the <loc href="#element-vocab-type-metadata">Metadata Module</loc>.</p>
 </def>
 </gitem>
+<gitem id="terms-module">
+<label>[module]</label>
+<def>
+<p>A collection of element types, attributes, features, or the specification(s) of such entities. Modules may be defined
```

Comment:
I'm not sure if any change is needed here, but in thinking about existing documents that could be construed as modules, for example those published by ARIB, EBU, SMPTE et al, it seems that it is common practice for those documents also to form some kind of profile, in other words, in addition to defining syntax and semantics of entities, they also define constraints on documents and/or processors.

It seems messy to make profile definition documents also be external modules, but perhaps we should add an informative note that it is possible for a single document to be (or contain) both a modules and profile definitions simultaneously, and clarify the difference between the two.

Reply:
Why are you making reference to "profiles" in this context? The definition of module does not refer to profile, and is effectively orthogonal to the profile system. A Module is a collection of "specified functionality". A Profile is a specification of designated features (or extensions) to be supported by a processor or used (or not) in a document.

Reply:
Because I think that point (with which I agree) may not be obvious to all readers, for whom TTML _already_ seems complicated without introducing a whole new thing. Further, the fact that modules and profiles are orthogonal means they can be combined into a single document without harm; from a logical perspective the two things have very different functions.

Reply:
We should not say anything about profiles in the definition of [module]. We do not define any profiles as a module, and profiles do not define functionality, so they are not a candidate to be defined as a module in the first place.

Reply:
On the contrary, some profile specifications _do_ additionally define functionality, which they typically then reference in the profile, thus such specifications fulfil the role or _profile_ and the role of _module_ simultaneously. IMSC does this for example. Informatively clarifying that this is possible and reasonable within the constructs introduced seems worthwhile to me.

Reply:
A profile does not define functionality. It refers to it. If it happens that a document (specification) happens to define both a profile (i.e., a TTML Profile Document or equivalent) and some functionality, then the functionality it defines could be reasonable labelled as one or more modules and registered in the module registry by making reference to the defining document (specification). However, the profile itself, i.e., the associated TTML Profile Document (or equivalent) is not functionality in its own right, and does not fall under the definition of [module]. As such, it would be incorrect to use the term _profile_ as a unit of functionality and include that in the definition of [module].

Reply:
Note: I would be willing to add a note indicating that a document (specification) that calls itself a profile may define functionality that can be labeled and registered as a module (for the defined functionality only, and not the profile as such).

Reply:
Added note in https://github.com/w3c/ttml3/pull/30/commits/c12c80df11a96101ddd5b3fd3454530a718d1741.

Reply:
Thanks, that works for me.